resources/views/index.html

```diff
@@ -5,22 +5,115 @@
+                    </style>
+                    <g id="Слой-1" serif:id="–°–ї–Њ–є 1">
+                        <g>
+                            <path class="st0"
+                                  d="M481.395,508.535c-5.4-71.26-28.382-117.45-39.574-143.628c-11.187-26.174-28.387-50.675-28.398-50.637    c-0.033,0.112-15.486,83.388-43.271,143.639c-4.638,10.064-10.797,22.626-17.013,32.744l0.002,0.001    c-19.183,33.866-47.013,66.265-63.604,99.105c-16.449,32.546-19.501,64.78-17.97,101.427    c1.543,36.85,10.054,72.992,18.227,108.816c30.81-6.647,57.628-18.023,80.825-32.563c61.05-38.274,97.939-99.492,108.423-165.444    c0,0,0.511-2.678,0.738-5.945C484.509,545.078,482.945,528.996,481.395,508.535z"></path>
+                            <path class="st1"
+                                  d="M481.395,508.535c-5.4-71.26-28.382-117.45-39.574-143.628c-11.187-26.174-28.387-50.675-28.398-50.637    c0,0-0.004,0.017-0.004,0.018c0.001-0.011,0.004-0.023,0.004-0.023l-4.106-6.105c-90.029-126.38-262.69-189.479-408.57-130.996    c-7.024,88.584,34.046,241.004,183.875,283.517c60.572,18.634,109.076,13.803,168.521,29.966    c-0.003,0.002-0.003,0.004-0.005,0.005c0,0,60.424,21.058,95.576,52.638c15.813,14.202,31.646,32.891,30.851,55.121    C484.574,545.505,482.97,529.32,481.395,508.535z"></path>
+                            <g transform="matrix(1,0,0,1,233.564,496.875)">
+                                <path class="st2"
+                                      d="M182.406-256.769c-21.289-62.282-12.267-104.016,26.679-162.096c18.575-27.711,50.648-59.854,78.756-78.01     C401.25-425.808,443.183-293.436,401.476-170.5c-30.354,89.45-58.833,126.955-130.84,218.99     c8.393-98.569-26.297-166.901-60.359-242.831C201.61-213.659,189.622-235.652,182.406-256.769"></path>
+                            </g>
+                            <g transform="matrix(1,0,0,1,245.403,498.558)">
+                                <path class="st3"
+                                      d="M234.164,99.853c0.795-22.23-15.038-40.919-30.852-55.121c-35.152-31.579-95.574-52.637-95.574-52.637     c6.217-10.118,12.375-22.681,17.012-32.745c27.786-60.252,43.239-143.526,43.272-143.639c0.011-0.038,17.21,24.464,28.398,50.637     c11.193,26.179,34.175,72.368,39.574,143.628C237.567,30.761,239.171,46.947,234.164,99.853z"></path>
+                            </g>
+                        </g>
+                    </g>
+                </svg>
+            </a>
+        </div>
+        <div class="yii-debug-toolbar__block">
+            <a href="#" title="">PHP
+                <span class="yii-debug-toolbar__label" data-key="phpVersion"></span>
+            </a>
+        </div>
+        <div class="yii-debug-toolbar__block">
+            <a href="#" title="Response status: 200">Status
+                <span class="yii-debug-toolbar__label yii-debug-toolbar__label_success" data-key="responseStatusCode">200</span>
+            </a>
+            <a href="#" title="Total request processing time was 15.17 ms">Time
```

Comment:
15.17 ms? Is there a moment when this is visible before data is loaded? If so, it is better to use something more meaningless like "Loading" or just zero.

Reply:
Leftover :)